File: app/src/main/java/com/babestudios/companieshouse/data/model/officers/Identification.java
package com.babestudios.companieshouse.data.model.officers;


import com.google.gson.annotations.SerializedName;

public class Identification {
	@SerializedName("place_registered")
	public String placeRegistered;
	@SerializedName("identification_type")
	public String identificationType;
	@SerializedName("registration_number")
	public String registrationNumber;
}

File: data/src/main/java/com/babestudios/companyinfo/data/model/officers/Identification.kt
package com.babestudios.companyinfo.data.model.officers


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Identification(
		@SerializedName("place_registered")
		var placeRegistered: String? = null,
		@SerializedName("identification_type")
		var identificationType: String? = null,
		@SerializedName("registration_number")
		var registrationNumber: String? = null
) : Parcelable
